class/class.2.1/test_script.ps1

```diff
@@ -0,0 +1,60 @@
+# test_script.ps1
+
+function ClearStatsFile {
+    "" > user.txt
+    Write-Host "user.txt cleared."
+}
+
+
+function RunTest {
```

Comment:
Minor: We can do it this way but in general for more complex test you may need some basic functions like checkOutput or checkFileContent and so on.  but for the current needs your implementation is ok